src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs

```diff
@@ -58,6 +58,11 @@ public interface IMediaElement : IView
 	/// <remarks>If media is paused, stopped or has completed playing, the display will turn off.</remarks>
 	bool ShouldKeepScreenOn { get; set; }
 
+	/// <summary>
+	/// Gets or sets if audio should be muted.
+	/// </summary>
+	bool ShouldMute { get; set; }
```

Comment:
Let's add a `<remarks` here about how this impacts volume control.

Something like this (caveat: I'm unsure if this statement is accurate to how we handle `Volume` vs `ShouldMute`)

```suggestion
	/// <summary>
	/// Gets or sets if audio should be muted.
	/// </summary>
	/// <remarks>When <see cref="ShouldMute"/> is <see langword="true" />, the value of <see cref="Volume"/> is ignored.</remarks>
	bool ShouldMute { get; set; }
```